src/RemoteFileAPI/Remote.ts

```diff
@@ -52,10 +68,21 @@ export class Remote {
       if (event.code === eventCodeWhenIntentionallyStoppingConnection) {
         return;
       }
-      SnackbarEvents.emit(`Remote API connection closed. Code: ${event.code}.`, ToastVariant.WARNING, 2000);
+
+      //Printing a connetion error alongside the connection closed warning is both redunant and confusing
+      if (successfullyConnected)
+        SnackbarEvents.emit(`Remote API connection closed. Code: ${event.code}.`, ToastVariant.WARNING, 2000);
+
       if (Settings.RemoteFileApiReconnectionDelay > 0) {
         setTimeout(() => {
-          this.startConnection();
+          if (!autoConnectAttempt || successfullyConnected) {
+            SnackbarEvents.emit(`Attempting to auto connect Remote API`, ToastVariant.WARNING, 2000);
+          }
```

Comment:
```suggestion
          if (autoConnectAttempt === 1 && Settings.RemoteFileApiReconnectionDelay > 0) {
            SnackbarEvents.emit(`Attempting to auto connect Remote API`, ToastVariant.WARNING, 2000);
          }
```
To do my suggestion of having this pop up immediately when you first enable auto-connection, this would be pulled out to the main part instead of being inside onClose.

Reply:
My suggestion here was for pulling it out to the main section instead of it being inside onClose. Watching the video, the timing with it like this also seems fine, but if you want to keep it here the `Settings.RemoteFileApiReconnectionDelay > 0` is redundant and should be removed.

Reply:
I applied this myself so we can get this submitted.